test/fixtures/array-ti.ts

```diff
@@ -0,0 +1,11 @@
+import * as t from "ts-interface-checker";
+// tslint:disable:object-literal-key-quotes
+
+export const IMyArrayContainer = t.iface([], {
+  "myArray": t.array("number"),
```

Comment:
It would be great to add one more complex example case like `myArray2: Array<{foo: number; bar: string}>`.